gui/velociraptor/src/components/users/user-label.jsx

```diff
@@ -403,6 +403,7 @@ export default class UserLabel extends React.Component {
     setSettings = (options) => {
         // Set the ACE theme according to the theme so they match.
         let ace_options = JSON.parse(this.context.traits.ui_settings || "{}");
+        ace_options.fontSize = "16px";
```

Comment:
This will override the user's choices in all cases - we should add it to the default settings here
https://github.com/Velocidex/velociraptor/blob/40c5a2283e9719ee3e9cd84c0d650d8400b28464/services/users/users.go#L61

Reply:
But that already says 20, so it doesn't seem to be working. Maybe it needs a "px" unit specifier added?

Reply:
Also the ACE editor brings it's own CSS and I've read that ace_options.fontSize is the right way to set it, even if it needs to be set somewhere else. I don't think the users.go setting will override ACE's default.